<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite thread-count="3" parallel="tests" name="Failed suite [Default suite]" guice-stage="DEVELOPMENT">
  <test thread-count="3" name="Default test-1(failed)">
    <parameter name="platform" value="1"/>
    <classes>
      <class name="testObjects.LoginErrorValidationTest">
        <methods>
          <include name="setUp"/>
          <include name="loginErrorValidation"/>
          <include name="tearDown"/>
        </methods>
      </class> <!-- testObjects.LoginErrorValidationTest -->
    </classes>
  </test> <!-- Default test-1(failed) -->
  <test thread-count="3" name="Default test-0(failed)">
    <parameter name="platform" value="0"/>
    <classes>
      <class name="testObjects.LoginErrorValidationTest">
        <methods>
          <include name="setUp"/>
          <include name="loginErrorValidation"/>
          <include name="tearDown"/>
        </methods>
      </class> <!-- testObjects.LoginErrorValidationTest -->
    </classes>
  </test> <!-- Default test-0(failed) -->
  <test thread-count="3" name="Default test-2(failed)">
    <parameter name="platform" value="2"/>
    <classes>
      <class name="testObjects.LoginErrorValidationTest">
        <methods>
          <include name="setUp"/>
          <include name="loginErrorValidation"/>
          <include name="tearDown"/>
        </methods>
      </class> <!-- testObjects.LoginErrorValidationTest -->
    </classes>
  </test> <!-- Default test-2(failed) -->
</suite> <!-- Failed suite [Default suite] -->
